FrontEnd/src/pages/Extensions/ExtensionDetailScreen.tsx

```diff
@@ -30,6 +30,7 @@ export function ExtensionDetailScreen() {
       <h2>Extension: {extensionDetails.name}</h2>
       <p>Area: {extensionDetails.area}</p>
       <p>Type: {extensionDetails.type}</p>
+      <p>Language: {extensionDetails.languages}</p>
```

Comment:
Maybe change it to Language(s) to dignify that there could be multiple languages.